bitcoin/wallet.go

```diff
@@ -213,6 +215,34 @@ func (w *BitcoinWallet) Transactions() ([]wi.Txn, error) {
 
 func (w *BitcoinWallet) GetTransaction(txid chainhash.Hash) (wi.Txn, error) {
 	txn, err := w.db.Txns().Get(txid)
+	if err == nil {
+		tx := wire.NewMsgTx(1)
```

Comment:
Is it better to use the existing constant `wire.TxVersion` (https://github.com/btcsuite/btcd/blob/5bda5314ca9549a589e63d7b2e6104492a0d5328/wire/msgtx.go#L18)?

Reply:
I think it might be better to use 1 as the lib may change under us. We only want to use version 1.